### ISSUE TITLE
RSVP closure reminder

### ISSUE THREAD

**Issue by raybradley** [scoutplan/scoutplan#254]
Send email/SMS 24 (12?) hours before RSVP closure. Depends on #253

**Comment by raybradley** [comment]
For each future, published event, send a note to all members with incomplete responses that the event will be closing soon

**Comment by raybradley** [comment]
This can probably just be folded into the existing RSVP nag by changing the wording to emphasize impending closure.
